decidim-pages/app/views/decidim/pages/application/show.html.erb

```diff
@@ -1,5 +1,4 @@
 <% add_decidim_meta_tags({
-  title: translated_attribute(@page.title),
```

Comment:
Static pages do not have this problem: see https://decidim.barcelona/pages/accessibility, title is "Accessibilitat - decidim.barcelona".

This title is set [in `layouts/decidim/participatory_process`](https://github.com/AjuntamentdeBarcelona/decidim/blob/7c1a3d9a346124649bfe0db21578ebf627d895f9/decidim-core/app/views/layouts/decidim/participatory_process.html.erb#L1).